src/terminal.cpp

```diff
@@ -2755,7 +2755,7 @@ uint8_t Terminal::consumeParamsAndGetCode(int * params, int * paramsCount, bool
       #endif
 
       // reset non specified parameters
-      while (p < params + FABGLIB_MAX_CSI_PARAMS)
+      while (p < params + FABGLIB_MAX_CSI_PARAMS-1)
```

Comment:
Why not fill/memset the whole params array with 0 at the beginning? Seems simpler to me and will simplify the code in the while(true) loop.

Reply:
I've done it as you suggested.